integration-tests/deployment/ccip/add_chain_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
-func TestAddChainInbound(t *testing.T) {
+func TestAddChain(t *testing.T) {
```

Comment:
If you are enabling the chain both way - let's send message in other direction to see whether that's successful